src/textual/app.py

```diff
@@ -508,6 +518,7 @@ async def process_messages(self) -> None:
         active_app.set(self)
         log("---")
         log(f"driver={self.driver_class}")
+        log(f"uvloop installed: {self.__class__.__uvloop_installed!r}")
```

Comment:
can be useful to know whether we're using uvloop or not?

Reply:
How about instead of this class attribute, we print the running loop? I would think that the repr would indicate if it is uvloop or not.

Reply:
It would be better indeed! :+1: 
The repr is not super easy to read in the case of a "native" asyncio loop, but it's useful for sure :slightly_smiling_face: 
 - native loop, for my Unix/Linux system: `<_UnixSelectorEventLoop running=True closed=False debug=False>`
 - uvloop's repr: `<uvloop.Loop running=True closed=False debug=False>`